cachito/web/migrations/env.py

```diff
@@ -23,7 +24,10 @@
 # target_metadata = mymodel.Base.metadata
 from flask import current_app
 
-config.set_main_option("sqlalchemy.url", current_app.config.get("SQLALCHEMY_DATABASE_URI"))
+if current_app.config["SQLALCHEMY_DATABASE_URI"] is None:
```

Comment:
Keep or drop this check? It was originally added while using get() instead of direct indexing, and it looks like it already might be covered in: https://github.com/containerbuildsystem/cachito/blob/f59adc16eba2166d43345a4c4da71b80fee261ce/cachito/web/config.py#L128